operacje_na_listach/Take_n _tems_from_the_list.py

```diff
@@ -0,0 +1,9 @@
+def take(list, items):
```

Comment:
`take(list, items)` is not that readable - `take(list_, number_of_items)` would be better